src/SixLabors.Fonts/TextOptions.cs

```diff
@@ -88,20 +87,12 @@ public float Dpi
         }
 
         /// <summary>
-        /// Gets or sets the width of the tab. Measured as the distance in spaces.
-        /// <para/>
-        /// Defaults to 4.
+        /// Gets or sets the width of the tab. Measured as the distance in spaces (U+0020).
         /// </summary>
-        public float TabWidth
-        {
-            get => this.tabWidth;
-
-            set
-            {
-                Guard.MustBeGreaterThanOrEqualTo(value, 0, nameof(this.TabWidth));
-                this.tabWidth = value;
-            }
-        }
+        /// <remarks>
+        /// If value is -1 then the font default tab width is used.
+        /// </remarks>
+        public float TabWidth { get; set; } = -1F;
```

Comment:
Our original implementation wasn't accurate and was actually multiplying the tab advance. This now really uses spaces.